app/supplementary_data.py

```diff
@@ -0,0 +1,89 @@
+import json
```

Comment:
I don't think this should be in app root. I suggest moving to utilities or introduce a `services/` folder. This is where CIR can also go.

Reply:
Added a services folder 👍